src/Raven.Server/Documents/Subscriptions/SubscriptionConnectionsState.cs

```diff
@@ -634,5 +634,50 @@ private SubscriptionConnectionsState(DocumentsStorage storage, SubscriptionState
         }
 
         public static SubscriptionConnectionsState CreateDummyState(DocumentsStorage storage, SubscriptionState state) => new(storage, state);
+
+        private long _lastNoopAckTicks;
+        private Task _lastNoopAckTask = Task.CompletedTask;
+
+        public Task SendNoopAck()
+        {
+            if (ClusterCommandsVersionManager.CurrentClusterMinimalVersion >= 53_000)
+            {
+                if (IsConcurrent)
+                {
+                    var localLastNoopAckTicks = Interlocked.Read(ref _lastNoopAckTicks);
+                    var nowTicks = DateTime.Now.Ticks;
+                    if ((nowTicks - localLastNoopAckTicks) / TimeSpan.TicksPerMillisecond < SubscriptionConnection.WaitForChangedDocumentsTimeoutInMs)
+                    {
+                        return _lastNoopAckTask;
+                    }
+
+                    var currentLastNoopAckTicks = Interlocked.CompareExchange(ref _lastNoopAckTicks, nowTicks, localLastNoopAckTicks);
+                    if (currentLastNoopAckTicks != localLastNoopAckTicks)
```

Comment:
Why do you do time based checks only on concurrent subscriptions? 
Because you want to avoid ack per connection?

Reply:
yes, for concurrent subscription we want to do the ack command once per all connections, so we do it once per state.
for regular subscription there is only 1 connection so no need to do the time based checks.
you can see here the result of the optimization: 
https://issues.hibernatingrhinos.com/issue/RavenDB-18883/Create-timeout-configuration-for-Acknowledge-subscription-command-when-the-CV-wasnt-chagned#focus=Comments-67-406964.0-0